src/abstractdataframe/abstractdataframe.jl

```diff
@@ -390,21 +390,23 @@ describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissi
 * `io` : optional output descriptor
 * `stats::AbstractVector{Symbol}`: the summary statistics to report. Allowed 
   fields are `:mean`, `:std`, `:min`, `:q25`, `:median`, `:q75`, `:max`, `:eltype`, 
-  `:nunique`, and `:nmissing`
+  `:nunique`, `:first`, `:last`, `:nmissing`, and `:all`
 
 **Result**
 
 * A `DataFrame` where each row represents a variable and each column a summary statistic.
 
 **Details**
 
-If the column's base type derives from `Number`, compute the mean, standard 
-deviation, minimum, first quantile, median, third quantile, and maximum. If 
-a column is not numeric, these statistics are populated with `nothing`s. 
+For `Real` columns, compute the mean, standard eviation, minimum, first quantile, median, 
+third quantile, and maximum. If a column does not derive from `Real`, `describe` will 
+attempt to calculate all statistics, using `nothing` as a fall-back in the case of an error.
 
 When `stats` contains `:nunique`, `describe` will report the 
-number of unique values in a column. If a column's base type derives from `Number`,
-`:nunique` will return `nothing`s.
+number of unique values in a column. If a column's base type derives from `Real`,
+`:nunique` will return `nothing`s. 
+
+When `stats` equals `:all`, `describe` will report all summary statistics.
```

Comment:
Better mention this when documenting `stats`.

Reply:
okay hopefully its ready to merge now.